### PR TITLE
Deprecate some useless configs

### PR DESCRIPTION
**Pull request by lance6716**
### First-time contributors' checklist 

- [ ] I've signed [**Contributor License Agreement**](https://cla-assistant.io/pingcap/docs-cn) that's required for repo owners to accept my contribution.

### Which TiDB version(s) do your changes apply to? (Required)

**Tips for choosing the affected version(s):**

By default, **CHOOSE MASTER ONLY** so your changes will be applied to the next TiDB major or minor releases. If your PR involves a product feature behavior change or a compatibility change, **CHOOSE THE AFFECTED RELEASE BRANCH(ES) AND MASTER**.

For details, see [tips for choosing the affected versions (in Chinese)](https://github.com/pingcap/docs-cn/blob/master/CONTRIBUTING.md#版本选择指南).

- [x] master (the latest development version)
- [x] v6.1 (TiDB 6.1 versions)
- [ ] v6.0 (TiDB 6.0 versions)
- [ ] v5.4 (TiDB 5.4 versions)
- [ ] v5.3 (TiDB 5.3 versions)
- [ ] v5.2 (TiDB 5.2 versions)
- [ ] v5.1 (TiDB 5.1 versions)
- [ ] v5.0 (TiDB 5.0 versions)
- [ ] v4.0 (TiDB 4.0 versions)

### What is the related PR or file link(s)?

- This PR is translated from:
- Other reference link(s): https://github.com/pingcap/tiflow/pull/5223

### Do your changes match any of the following descriptions?

- [ ] Delete files
- [ ] Change aliases
- [ ] Need modification after applied to another branch 
- [ ] Might cause conflicts after applied to another branch

### CODE REVIEW

File: dm/relay-log.md [hunk 1]
```diff
@@ -10,6 +10,10 @@ DM (Data Migration) 工具的 relay log 由若干组有编号的文件和一个
 
 在启用 relay log 功能后，DM-worker 会自动将上游 binlog 迁移到本地配置目录（若使用 TiUP 部署 DM，则迁移目录默认为 `<deploy_dir> / <relay_log>`）。本地配置目录 `<relay_log>` 的默认值是 `relay-dir`，可在[上游数据库配置文件](/dm/dm-source-configuration-file.md)中进行修改。自 v5.4.0 版本起，你可以在 [DM-worker 配置文件](/dm/dm-worker-configuration-file.md)中通过 `relay-dir` 配置本地配置目录，其优先级高于上游数据库的配置文件。
 
+> **警告：**
+>
+> 上有数据库配置文件中的 `relay-dir` 在 v6.1 版本已经标记为弃用，将会在未来版本移除。在相关命令中您会看到 `` `relay-dir` in source config will be deprecated soon, please use `relay-dir` in worker config instead`` 的提示。
```
Comment (lance6716) [inline dm/relay-log.md:15]:
```suggestion
> 上游数据库配置文件中的 `relay-dir` 在 v6.1 版本中标记为弃用，在未来版本可能会被移除。相关命令的输出中您会看到 `` `relay-dir` in source config will be deprecated soon, please use `relay-dir` in worker config instead`` 的提示。
```

File: dm/relay-log.md [hunk 2]
```diff
@@ -131,6 +135,10 @@ Relay log 迁移的起始位置由如下规则决定：
 > Please use `start-relay` to specify which workers should pull relay log of relay-enabled sources.
 > ```
 
+> **警告：**
+>
+> 该启动方式在 v6.1 版本中标记为弃用，在未来版本可能会被移除。在这些命令的输出中您会看到 `start-relay/stop-relay with worker name will be deprecated soon. You can try stopping relay first and use start-relay without worker name instead` 的提示。
```
Comment (lance6716) [inline dm/relay-log.md:140]:
```suggestion
> 该启动方式在 v6.1 版本中标记为弃用，在未来版本可能会被移除。相关命令的输出中您会看到 `start-relay/stop-relay with worker name will be deprecated soon. You can try stopping relay first and use start-relay without worker name instead` 的提示。
```

File: dm/relay-log.md [hunk 3]
```diff
@@ -10,6 +10,10 @@ DM (Data Migration) 工具的 relay log 由若干组有编号的文件和一个
 
 在启用 relay log 功能后，DM-worker 会自动将上游 binlog 迁移到本地配置目录（若使用 TiUP 部署 DM，则迁移目录默认为 `<deploy_dir> / <relay_log>`）。本地配置目录 `<relay_log>` 的默认值是 `relay-dir`，可在[上游数据库配置文件](/dm/dm-source-configuration-file.md)中进行修改。自 v5.4.0 版本起，你可以在 [DM-worker 配置文件](/dm/dm-worker-configuration-file.md)中通过 `relay-dir` 配置本地配置目录，其优先级高于上游数据库的配置文件。
 
+> **警告：**
+>
+> 上游数据库配置文件中的 `relay-dir` 在 v6.1 版本中标记为弃用，在未来版本可能会被移除。相关命令的输出中您会看到 `` `relay-dir` in source config will be deprecated soon, please use `relay-dir` in worker config instead`` 的提示。
```
Comment (ran-huang) [inline dm/relay-log.md:15]:
```suggestion
> 上游数据库配置文件中的 `relay-dir` 在 v6.1 版本中标记为弃用，在未来版本可能会被移除。相关命令的输出中您会看到如下提示： `` `relay-dir` in source config will be deprecated soon, please use `relay-dir` in worker config instead``。
```

File: dm/task-configuration-file-full.md [hunk 4]
```diff
@@ -22,7 +22,7 @@ aliases: ['/docs-cn/tidb-data-migration/dev/task-configuration-file-full/','/doc
 ## ********* 基本信息配置 *********
 name: test                      # 任务名称，需要全局唯一
 task-mode: all                  # 任务模式，可设为 "full" - "只进行全量数据迁移"、"incremental" - "Binlog 实时同步"、"all" - "全量 + Binlog 实时同步"
-is-sharding: true               # 该配置项从 DM v2.0.0 起弃用，其功能被 `shard-mode` 取代，建议使用 `shard-mode` 代替 `is-sharding`
+# is-sharding: true             # 该配置项从 DM v2.0.0 起弃用，其功能被 `shard-mode` 取代，建议使用 `shard-mode` 代替 `is-sharding`
```
[the remark this answers is not in the document]
Comment (lance6716) [reply dm/task-configuration-file-full.md:25]: /cc @sunzhaoyang

File: dm/relay-log.md [hunk 5]
```diff
@@ -131,6 +135,10 @@ Relay log 迁移的起始位置由如下规则决定：
 > Please use `start-relay` to specify which workers should pull relay log of relay-enabled sources.
 > ```
 
+> **警告：**
+>
+> 该启动方式在 v6.1 版本中标记为弃用，在未来版本可能会被移除。相关命令的输出中您会看到 `start-relay/stop-relay with worker name will be deprecated soon. You can try stopping relay first and use start-relay without worker name instead` 的提示。
```
[the remark this answers is not in the document]
Comment (lance6716) [reply dm/relay-log.md:140]: 这三个 tab 对应的行为是不一样的，之后这个 tab 的行为被弃用了

File: dm/task-configuration-file-full.md (hunk 4 above)
Comment (sunzhaoyang) [inline dm/task-configuration-file-full.md:25]:
```suggestion
```

Comment (sunzhaoyang) [reply dm/task-configuration-file-full.md:25]: agree

**Review by sunzhaoyang — APPROVED** [review]
rest LGTM

File: dm/relay-log.md (hunk 5 above)
Comment (ran-huang) [inline dm/relay-log.md:140]:
```suggestion
> 该启动方式在 v6.1 版本中标记为弃用，在未来版本可能会被移除。相关命令的输出中您会看到如下提示：`start-relay/stop-relay with worker name will be deprecated soon. You can try stopping relay first and use start-relay without worker name instead`。
```

**Review by ran-huang — APPROVED** [review]

**Pull request merged**